src/utils/EventManager.ts

```diff
@@ -12,6 +12,9 @@ module n3Charts.Utils {
       'over',   // on mouse over a data point or column
       'out',   // on mouse out of a data point or column
       'click',   // on click on a data point or column
+      'legend-click',   // on click on a legend item
+      'legend-over',   // on mouse over on a legend item
+      'legend-out',   // on mouse out on a legend item
```

Comment:
Same as my other comment on events names, I think we need to wait until we see a broader picture for the events...